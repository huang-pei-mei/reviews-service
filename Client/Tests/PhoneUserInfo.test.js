import 'jsdom-global/register';
import React from 'react';
import { shallow, mount, render } from 'enzyme';
import "@babel/polyfill";
import PhoneUserInfo from '../src/Components/phoneUserInfo.js';
import mockSuccessAjax from './mockSuccessAjax';
import successData  from './successData.json';
import renderer from 'react-test-renderer'

// it ('should match snapShot', () => {
//   let review = mockSuccessAjax(successData);
//   review = review.slice(0, 1);
//   const tree = renderer.create(<PhoneUserInfo review={review} />).toJSON();

//   expect(tree).toMatchSnapshot();
// });

it ('should render phoneUserinfo', () => {
  let review = mockSuccessAjax(successData);
  review = review.slice(0, 1);
  const phoneUser = shallow(<PhoneUserInfo review={review} />);
  expect(phoneUser.find('.phoneUserInfo').exists()).toBe(true);
})