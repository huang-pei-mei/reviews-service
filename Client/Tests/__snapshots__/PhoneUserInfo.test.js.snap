// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapShot 1`] = `
<div
  className="phoneUserInfo"
>
  <div
    className="By"
  >
    By
  </div>
  <div
    className="phoneUserName"
  />
  <div
    className="phoneOnDate"
  >
    on 
    05-13-2021
  </div>
</div>
`;
